SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -273,6 +294,34 @@ void ASpatialFunctionalTest::StartTest()
 	StartStep(0);
 }
 
+void ASpatialFunctionalTest::CallRunTest(const TArray<FString>& Params)
+{
+	if (HasAuthority() == false)
```

Comment:
Maybe add some comments as to why this is needed?

Reply:
I think the header file has a comment explaining why we added this function? Do you think that is not clear enough?

Reply:
Hmm I think putting the comment here is preferable since its an override, the extra work its doing makes a lot more sense in context.

Reply:
Oky, will do!